src/legend_data_monitor/cuts.py

```diff
@@ -19,10 +20,210 @@ def cut_k_lines(data):
     return data[(data[energy] > 1430) & (data[energy] < 1575)]
 
 
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
@sagitta42 some cleaning here :') 

If the cut itself could be passed to the function, then it is done.
ps The cuts are applied one by one